lib/ansible/modules/monitoring/zabbix/zabbix_host.py

```diff
@@ -340,8 +325,7 @@ def update_host(self, host_name, group_ids, status, host_id, interfaces, exist_i
             if self._module.check_mode:
                 self._module.exit_json(changed=True)
             parameters = {'hostid': host_id, 'groups': group_ids, 'status': status, 'tls_connect': tls_connect,
-                          'tls_accept': tls_accept, 'ipmi_authtype': ipmi_authtype, 'ipmi_privilege': ipmi_privilege,
```

Comment:
Hi @eikef . Can you apply same changes for the add_host as well?

Reply:
Thanks for spotting that. Did so now :)